packages/docker/Dockerfile.nextjs

```diff
@@ -12,17 +12,23 @@ ENV NEXT_TELEMETRY_DISABLED 1
 
 WORKDIR /app
 
+FROM base as prepare
+
+COPY package*.json /app
+COPY /$WORKSPACE/package.json /app/$WORKSPACE/package.json
+COPY --parents /packages/*/package.json /app
+# reset app version to 1.0.0 to avoid cache busting after every release
+RUN npm version --allow-same-version 1.0.0 -w $WORKSPACE --no-workspaces-update
```

Comment:
question (non-blocking): will that affect the version of the app once deployed?

Reply:
No, it's just to have a stable version to run npm ci and make sure docker caches this. Later versions are overwritten back to actual values at these lines -> https://github.com/akash-network/console/pull/1031/files#diff-a75f5b03c61fa0c6a799dea7b23933d2cee82c2baf971036b727cac4ccd162dcR33-R34

Reply:
though this requires a bit more investigation. The issue is that `npm version` installs dependencies if they are not installed, could you imagine that?)) 

Addition of `--no-workspaces-update` disables this behavior but it also does not update package-lock in this case. But I need to update it as well. Without this option `npm version` updates both json files but also installs dependencies 😄

Reply:
the fix is to use `npm install --package-lock-only` which updates only package lock file and do not install dependencies